server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -1785,6 +1783,13 @@ export interface components {
        */
       reviewDate?: string
     }
+    GetGoalsResponse: {
+      /**
+       * @description A List of zero or more Goals.
+       * @example null
+       */
+      goals: components['schemas']['GoalResponse'][]
+    }
```

Comment:
I had to manually create this as there is something wrong with the swagger being generated for this endpoint. I spent a bit of time trying to fix the swagger but can't see what's wrong with it. It is generating "object" rather than referencing the GetGoalsResponse schema.

Reply:
I wonder if you imported the spec from either before I made my change on Tuesday, or from your local branch where you hadn't pulled in my changes in some way?
The hosted swagger spec looks correct, so I think this should import and generate the code correctly. Am more worried that it doesnt TBH and what having made manual changes here might mean for future swagger spec imports. 
I think we should get to the bottom of this 🤔

Reply:
Nope because I had to re-import after you made your change from Array. It's like this if you run it up locally or inspect the JSON on dev.
I agree we should get to the bottom of it though. I spent some time on it earlier in the week and no matter what I changed I couldn't get it to put a reference instead of "object". I was hoping you'd know...